testing/sdk_auth.py

```diff
@@ -287,7 +287,8 @@ def list_principals(self, filter: str = "*") -> List[str]:
         parsed = res.json()
         print(parsed)
 
-        return parsed.get("data", [])
+        principals = parsed.get("principals", {})
```

Comment:
It seems you're changing the API in an incompatible way, but this function is not used anywhere yet, right?

Reply:
You are correct. The HTTP API has changed slightly in order to include the `checksum` in the response, in addition to the list of `principals`. 

This API is already in use by the KDC terraform provider and already up-to-date : https://github.com/mesosphere/data-services-terraform/blob/master/providers/terraform-provider-kdc/client/api.go#L31

In the scope of `dcos-commons` and the respective integration tests, this change has no impact.